docs/src/quickstart/build.rst

```diff
@@ -18,6 +18,8 @@ one may want to read more about
 There are several ways to build Cython code:
 
  - Write a setuptools ``setup.py``. This is the normal and recommended way.
+ - Run the ``cythonize`` command-line utility. This is a good approach for
```

Comment:
`cythonize` by default creates just C file. Can we mention here that the user should run `cythonize` with `-b` option?

Reply:
Yes `-i` also enables `-b` automatically:

```
  -b, --build           build extension modules using distutils
  -i, --inplace         build extension modules in place using distutils (implies -b)
```

Reply:
I've recommended `-i` just because it's what I usually do. I think the main thing to do here is just give a usable command to get them going. People can look up all the available options in elsewhere if they want.

Thanks for the suggestion!

Reply:
+1 to `-i`.